docs/content/developer/tutorials/sponsored-txs.mdx

```diff
@@ -0,0 +1,738 @@
+
+    let tx: TransactionData = bcs::from_bytes(&decoded_tx)
+        .map_err(|_| anyhow::anyhow!("Failed to deserialize transaction data"))?;
+    let sig_bcs = Signature::from_bytes(&decoded_sig)
+        .map_err(|_| anyhow::anyhow!("Failed to deserialize sponsor's signature"))?;
+
+    let intent_msg = IntentMessage::new(Intent::iota_transaction(), tx.clone());
+
+    let sender_addr = IotaAddress::from_str(sender_addr)
+        .map_err(|_| anyhow::anyhow!("Failed to parse sender address"))?;
+
+    let sender_sig = keystore
+        .sign_secure(&sender_addr, &tx, intent_msg.intent)
+        .map_err(|_| anyhow::anyhow!("Failed to sign transaction with sender's keystore"))?;
+
+    let signed_tx = iota_sdk::types::transaction::Transaction::from_generic_sig_data(
+        intent_msg.value,
+        vec![
+            iota_sdk::types::signature::GenericSignature::Signature(sender_sig),
+            iota_sdk::types::signature::GenericSignature::Signature(sig_bcs),
+        ],
+    );
+
+    let iota_testnet = IotaClientBuilder::default()
+        .build_testnet()
+        .await
+        .map_err(|_| anyhow::anyhow!("Failed to connect to IOTA testnet"))?;
+
+    let transaction_block_response = iota_testnet
+        .quorum_driver_api()
+        .execute_transaction_block(
+            signed_tx.clone(),
+            iota_sdk::rpc_types::IotaTransactionBlockResponseOptions::full_content()
+                .with_raw_input(),
+            iota_sdk::types::quorum_driver_types::ExecuteTransactionRequestType::WaitForLocalExecution,
+        )
+        .await
+        .map_err(|e| anyhow!("Failed to execute transaction: {:?}", e))?;
+
+    println!("{:?}", transaction_block_response);
+    Ok(())
+}
+```
+
+`construct_tx` function will try to decode the provided transaction bytes and sponsor's signature. Then, it will sign the transaction with the sender's keystore and send the transaction to the IOTA network and print the response.
+
+Now that we have the `tx_bytes` and `sponsor_sig` from the Gas Station server, we can use the `construct_tx` function to send the transaction to the IOTA network:
+
+```shell
+# Replace values with actual values from the Gas Server response. Also try to use a different address for the sponosor address that has no funds
+$ cargo run -- --keystore <KEYSTORE-PATH> --encoded-sig "AA15MUFqu8BQX3zWINWVewS5Ci0QeoCLp1AEffmZmzc8w+tnvVAS7gVkHRh61pdyNe+H4NdseUvwW9yLnchqkQ3p1vxEC3McO9IvqEI8BXaHBgCUFx3DXTO+QWmQKwj2uA==" --encoded-tx "AAACAAVNdXNpYwEBsFI25soGfj+hFLqxVY815ECX4AeKpoF2H6piIghYoXYOFAAAAAAAAAEBACBp6RyDMzUL32u9KZEmatM5knV9t69IKRrbWOelsOGqH3Nwb25zb3JlZF90cmFuc2FjdGlvbnNfcGFja2FnZXMKZnJlZV90cmlhbAACAQAAAQEAx9FYqbBcbf0HwjNknJ4PeDINBm5awOj1EB3lAK2ehOgBOeqaP5iJwfw9edY9l64pOnSFnQzguHbAkCEUceNwZ7sOFAAAAAAAACBC3cFbtd21gUyztwvt5k6hwgXZg7/nTKjNvH5WoXR0078pPO0lkxGM0jHxB/NBuxrZ2znNBJe/8p01VzDPTivC6AMAAAAAAABAS0wAAAAAAAA=" --sender-addr 0xc7d158a9b05c6dfd07c233649c9e0f78320d066e5ac0e8f5101de500ad9e84e8
```

Comment:
```suggestion
cargo run -- --keystore <KEYSTORE-PATH> --encoded-sig "AA15MUFqu8BQX3zWINWVewS5Ci0QeoCLp1AEffmZmzc8w+tnvVAS7gVkHRh61pdyNe+H4NdseUvwW9yLnchqkQ3p1vxEC3McO9IvqEI8BXaHBgCUFx3DXTO+QWmQKwj2uA==" --encoded-tx "AAACAAVNdXNpYwEBsFI25soGfj+hFLqxVY815ECX4AeKpoF2H6piIghYoXYOFAAAAAAAAAEBACBp6RyDMzUL32u9KZEmatM5knV9t69IKRrbWOelsOGqH3Nwb25zb3JlZF90cmFuc2FjdGlvbnNfcGFja2FnZXMKZnJlZV90cmlhbAACAQAAAQEAx9FYqbBcbf0HwjNknJ4PeDINBm5awOj1EB3lAK2ehOgBOeqaP5iJwfw9edY9l64pOnSFnQzguHbAkCEUceNwZ7sOFAAAAAAAACBC3cFbtd21gUyztwvt5k6hwgXZg7/nTKjNvH5WoXR0078pPO0lkxGM0jHxB/NBuxrZ2znNBJe/8p01VzDPTivC6AMAAAAAAABAS0wAAAAAAAA=" --sender-addr 0xc7d158a9b05c6dfd07c233649c9e0f78320d066e5ac0e8f5101de500ad9e84e8
```
Please don't add the $ in bash commands, it's annoying when the user copy pastes